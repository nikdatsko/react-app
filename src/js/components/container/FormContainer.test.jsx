import React from "react";
import ReactDOM from "react-dom";
import FormContainer from "./FormContainer";

describe("FormContainer", () => {
  it("renders without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(<FormContainer />, div);
    expect(div).toMatchSnapshot();
  });
});
