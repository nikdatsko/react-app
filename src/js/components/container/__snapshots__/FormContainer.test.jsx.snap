// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormContainer renders without crashing 1`] = `
<div>
  <form
    id="article-form"
  >
    <div
      class="form-group"
    >
      <label
        for="seo_title"
      >
        SEO title
      </label>
      <input
        class="form-control"
        id="seo_title"
        required=""
        type="text"
        value=""
      />
    </div>
  </form>
</div>
`;
